tests/sweep_framework/sweep_utils/utils.py

```diff
@@ -247,3 +248,45 @@ def complex_from_torch(torch_tensor, dtype, layout, memory_config, device):
         memory_config=memory_config,
     )
     return ttnn.complex_tensor(tt_real, tt_imag)
+
+
+def get_device_grid_size():
+    device_name = os.environ.get("ARCH_NAME", os.environ.get("TT_ARCH_NAME", "default")).lower()
+    assert device_name in ["wormhole_b0", "grayskull"]
+    if device_name == "grayskull":
+        y, x = 9, 12
+    else:
+        y, x = 8, 8
+
+    return y, x
+
+
+def get_sharded_config(input_shape, sharding_strategy, device_grid_size, shard_orientation):
+    assert sharding_strategy in ["block", "width", "height", "tensor_hw"]
+    assert shard_orientation in ["col_major", "row_major"]
+
+    if shard_orientation == "col_major":
+        orientation = ttnn.ShardOrientation.COL_MAJOR
+    else:
+        orientation = ttnn.ShardOrientation.ROW_MAJOR
+
+    if sharding_strategy == "block":
+        strategy = ttnn.ShardStrategy.BLOCK
+    elif sharding_strategy == "width":
+        strategy = ttnn.ShardStrategy.WIDTH
+    elif sharding_strategy == "height":
+        strategy = ttnn.ShardStrategy.HEIGHT
+    else:
+        strategy = ttnn.ShardStrategy.BLOCK
+
+    tensor_hw_as_shard_shape = True if sharding_strategy == "tensor_hw" else False
+
+    sharded_config = ttnn.create_sharded_memory_config(
```

Comment:
can you replace with `ttnn.create_sharded_memory_config_`?

Reply:
Done